doc/kernel/usermode/usermode.rst

```diff
@@ -9,6 +9,142 @@ are defined, and how memory may be managed to support user mode threads.
 For details on creating threads that run in user mode, please see
 :ref:`lifecycle_v2`.
 
+Threat Model
+============
+
+For threads that have either started initially, or dropped down to user mode,
+we aim to protect against the following:
+
+- Access to memories outside their own stack or program text not specifically
+  granted, or incorrect access to memories that have an incompatible policy,
+  such as attempting to write to a read-only area.
+
+- Access to device drivers or kernel objects not specifically granted,
+  with the permission granularity on a per object or per driver instance basis.
+
+- Making kernel or driver API calls with incorrect parameters that would
+  otherwise cause a crash or corruption of data structures private to the
+  kernel. This includes:
+
+  - Using the wrong kernel object type.
```

Comment:
Use a lowercase letter to start all these list items, and remove the trailing period.